src/toil_scripts/gatk_germline/germline_config.py

```diff
@@ -0,0 +1,120 @@
+import textwrap
+
+def generate_config():
+    return textwrap.dedent("""
+        # GATK Germline Pipeline configuration file
+        # This configuration file is formatted in YAML. Simply write the value (at least one space) after the colon.
+        # Edit the values in this configuration file and then rerun the pipeline
+        # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank.
+        ##############################################################################################################
+        # Required: Reference Genome URL (hg19.fa)
+        genome.fa:
+
+        # Optional: Reference Genome Index URL (hg19.fa.fai)
+        genome.fa.fai:
+
+        # Optional: Reference Genome Sequence Dictionary (hg19.dict)
+        genome.dict:
+
+        # Required: URL (1000G_phase1.indels.hg19.sites.fixed.vcf)
+        phase:
+
+        # Required: URL (Mills_and_1000G_gold_standard.indels.hg19.sites.vcf)
+        mills:
+
+        # Required: URL (dbsnp_132_b37.leftAligned.vcf URL)
+        dbsnp:
+
+        # Required: URL (hapmap_3.3.b37.vcf)
+        hapmap:
+
+        # Required: URL (1000G_omni.5.b37.vcf)
+        omni:
+
+        # Optional: (boolean) Run BWA on fastqs
+        run_bwa:
+
+        # Optional: If true, BWA sorts bam
+        sort: True
```

Comment:
Is there a reason we're exposing this as an option here? The pipeline downstream from BWA will fail on unsorted input.